x-pack/legacy/plugins/apm/mappings.json

```diff
@@ -1,42 +1,734 @@
+              }
+            }
+          },
+          "span": {
+            "properties": {
+              "1d": {
+                "type": "long"
+              },
+              "all": {
+                "type": "long"
+              }
+            }
+          },
+          "traces": {
+            "properties": {
+              "1d": {
+                "type": "long"
+              }
+            }
+          },
+          "transaction": {
+            "properties": {
+              "1d": {
+                "type": "long"
+              },
+              "all": {
+                "type": "long"
+              }
+            }
+          }
+        }
+      },
       "has_any_services": {
         "type": "boolean"
       },
+      "indices": {
+        "properties": {
+          "all": {
+            "properties": {
+              "total": {
+                "properties": {
+                  "docs": {
+                    "properties": {
+                      "count": {
+                        "type": "long"
+                      }
+                    }
+                  },
+                  "store": {
+                    "properties": {
+                      "size_in_bytes": {
+                        "type": "long"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "shards": {
+            "properties": {
+              "total": {
+                "type": "long"
+              }
+            }
+          }
+        }
+      },
+      "integrations": {
+        "properties": {
+          "ml": {
+            "properties": {
+              "all_jobs_count": {
+                "type": "long"
+              }
+            }
+          }
+        }
+      },
+      "retainment": {
+        "properties": {
+          "error": {
+            "properties": {
+              "ms": {
+                "type": "long"
+              }
+            }
+          },
+          "metric": {
+            "properties": {
+              "ms": {
+                "type": "long"
+              }
+            }
+          },
+          "onboarding": {
+            "properties": {
+              "ms": {
+                "type": "long"
+              }
+            }
+          },
+          "span": {
+            "properties": {
+              "ms": {
+                "type": "long"
+              }
+            }
+          },
+          "transaction": {
+            "properties": {
+              "ms": {
+                "type": "long"
+              }
+            }
+          }
+        }
+      },
       "services_per_agent": {
         "properties": {
-          "python": {
-            "type": "long",
-            "null_value": 0
+          "dotnet": {
+            "type": "long"
+          },
+          "go": {
+            "type": "long"
           },
           "java": {
-            "type": "long",
-            "null_value": 0
```

Comment:
I wonder if the removal of the "null_value" might cause any issues?

Feel free to dismiss this if this change is intended :)

Reply:
good catch - I don't think we need a fallback anymore but I'll put it there to prevent any backwards compat issues.